R/tbl-df.r

```diff
@@ -33,22 +33,22 @@ print.tbl_df <- print.tbl
 
 #' @export
 `[[.tbl_df` <- function(x, i, j, ..., exact = TRUE) {
-  if (missing(j)) {
-    colname <- i
-  } else {
-    colname <- j
-  }
   if (!exact) {
     warningc("exact ignored")
   }
+  if (missing(j)) {
+    return(unclass(x)[[i]])
```

Comment:
@lionel-: Is there a helper for `unclass(x))[[i]]` in *rlang*?

Reply:
Not yet, maybe you can use `.subset2()`?

Reply:
Of course, thanks!